catalyst_voices/lib/widgets/modals/voices_upload_file_dialog.dart

```diff
@@ -0,0 +1,371 @@
+import 'package:catalyst_voices/widgets/buttons/voices_filled_button.dart';
+import 'package:catalyst_voices/widgets/buttons/voices_outlined_button.dart';
+import 'package:catalyst_voices/widgets/indicators/voices_linear_progress_indicator.dart';
+import 'package:catalyst_voices/widgets/modals/voices_desktop_dialog.dart';
+import 'package:catalyst_voices/widgets/modals/voices_dialog.dart';
+import 'package:catalyst_voices_assets/catalyst_voices_assets.dart';
+import 'package:catalyst_voices_brands/catalyst_voices_brands.dart';
+import 'package:catalyst_voices_localization/catalyst_voices_localization.dart';
+import 'package:catalyst_voices_models/catalyst_voices_models.dart';
+import 'package:catalyst_voices_shared/catalyst_voices_shared.dart';
+import 'package:dotted_border/dotted_border.dart';
+import 'package:file_picker/file_picker.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_dropzone/flutter_dropzone.dart';
+import 'package:web/web.dart' as web;
+
+class VoicesUploadFileDialog extends StatefulWidget {
+  final String title;
+  final String itemNameToUpload;
+  final String? info;
+  final List<String>? allowedExtensions;
+  final Future<dynamic> Function(VoicesFile value)? onUpload;
```

Comment:
Either use generics for Future's type or just mark it as void. dynamic is never a good option